corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -602,6 +621,9 @@ cloudCare.AppView = Backbone.View.extend({
 
         return url;
     },
+    getSyncUrl: function () {
+        return "/webforms/player_proxy";
```

Comment:
actually we could post to another url and forward it to the `player_proxy` i think that's normally how we handle it. @czue any thoughts?